api/src/main/java/net/md_5/bungee/api/SkinConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package net.md_5.bungee.api;
+
+/**
+ * Represents a player's skin settings.
+ */
+public interface SkinConfiguration
```

Comment:
Can you please just document what skin settings are and how they are applied / what they can be used for.

On this basis of this I have absolutely no clue.
How does a player remove their left sleeve, what happens if they do, etc etc